.gitignore

```diff
@@ -25,3 +25,4 @@ __coverage__
 csak-timelog.json
 .idea/
 debug/
+package-lock.json
```

Comment:
I don't think this file should be ignored in this project... nor related to this PR